airflow/models/dagrun.py

```diff
@@ -349,13 +354,15 @@ def get_previous_scheduled_dagrun(self, session: Session = None):
         ).first()
 
     @provide_session
-    def update_state(self, session: Session = None) -> List[TI]:
+    def update_state(self, session: Session = None, handle_callback: Optional[bool] = True) -> List[TI]:
```

Comment:
```suggestion
    def update_state(self, session: Session = None, handle_callback: bool = True) -> List[TI]:
```

Optional + Boolean is technically a tristate variable, and since we have a default we don't need to allow this to be optional.

Reply:
fixed in https://github.com/astronomer/airflow/pull/913/commits/29af60d324509c46f983559b5ab6d702a42eebee